app/controllers/NotificationController.scala

```diff
@@ -172,5 +173,22 @@ class NotificationController @Inject()(private[controllers] val emailConnector:
             }
         }
     }
+
+  def checkReminderType(contactType: Option[ContactType], date: DateTime): (Option[ContactType], String) ={
```

Comment:
I think func name should be changed . It's not just checking the reminder TYpe.  May be something like `getContactTypeAndTemplateVersion`. what do you thnk?

Reply:
Yeah that sounds good will make the change